src/wasm-lib/kcl/src/std/shapes.rs

```diff
@@ -181,6 +181,9 @@ pub async fn circle_three_point(exec_state: &mut ExecState, args: Args) -> Resul
         tag = {docs = "Identifier for the circle to reference elsewhere."},
     }
 }]
+
+/// Similar to inner_circle, but needs to retain 3-point information in the
+/// path so it can be used for other features, otherwise it's lost.
```

Comment:
> Do you want this comment published on zoo.dev? I don't think users of KCL will know what to make of this, might be worth moving out of the doc comment.

You can use `//` to be a regular comment, not a doc comment.

Reply:
Yeah that's what I ended up doing. Great minds think alike!